src/main/java/com/powsybl/openloadflow/dc/DcLoadFlowEngine.java

```diff
@@ -42,7 +42,9 @@ public DcLoadFlowEngine(LfNetwork network, MatrixFactory matrixFactory, boolean
     }
 
     public DcLoadFlowEngine(Object network, DcLoadFlowParameters parameters, Reporter reporter) {
-        LfNetworkParameters lfNetworkParameters = new LfNetworkParameters(parameters.getSlackBusSelector(), false, false, false, false, parameters.getPlausibleActivePowerLimit(), false);
+        LfNetworkParameters lfNetworkParameters = new LfNetworkParameters(parameters.getSlackBusSelector(), false, false, false, false,
```

Comment:
Be careful here as we have in the code a `LfNetwork network = networks.get(0);` that says that we only deal with the first synchronous component. Do we want to do better here? Handle several synchronous components? Handle several connected components?

Reply:
Yes, I made some changes in the code : I replaced all the get(0) I could find in the code by the use of a filter on the network num on what is defined as the "main num". I use the constant parameter ComponentConstants.MAIN_NUM provided by the core for that.
This is just a begining while waiting to see what we do with several connected components more largely.